packages/locale/res/translations/it.json

```diff
@@ -0,0 +1,467 @@
+    ["Choose Password", "Scegli una password"],
+    ["Please choose a password!", "Si prega di scegliere una password!"],
+    ["Please repeat the password!", "Si prega di ripetere la password!"],
+    ["Password not repeated correctly!", "Password non ripetuta correttamente!"],
+    ["Choose A Field Type", "Scegli un tipo di campo"],
+    ["What kind of field you would like to add?", "Che tipo di campo vorresti aggiungere?"],
+    ["Enter Notes Here", "Inserisci le note qui"],
+    ["Enter Secret", "Inserisci chiave"],
+    ["Enter Password", "Inserisci password"],
+    ["Enter Value Here", "Inserisci valore qui"],
+    ["Enter Field Name", "Inserisci nome campo"],
+    ["Dash", "Trattino"],
+    ["Underscore", "Underscore"],
+    ["Slash", "Slash"],
+    ["Space", "Spazio"],
+    ["Generate Password", "Genera Password"],
+    ["passphrase", "passphrase"],
+    ["random string", "stringa casuale"],
+    ["Word Separator", "Separatore di caratteri"],
+    ["Language", "Lingua"],
+    ["words", "parole"],
+    ["length", "lunghezza"],
+    ["Use", "Usa"],
+    ["Discard", "Scarta"],
+    ["Are you sure you want to delete this group?", "Sei sicuro di voler eliminare questo gruppo?"],
+    ["Delete Group", "Elimina gruppo"],
+    ["Type 'DELETE' to confirm", "Digitare 'DELETE' per confermare"],
+    ["Enter Group Name", "Inserisci il nome del gruppo"],
+    ["Search...", "Ricerca..."],
+    ["Import Data", "Importa Dati"],
+    [
+        "IMPORTANT: Before importing, please make sure that your CSV data is structured according to Padlocks specific requirements!",
+        "IMPORTANTE: Prima di importare, assicurati che i tuoi dati CSV siano strutturati secondo i requisiti specifici di Padlock!"
+    ],
+    ["Learn More", "Scopri di più"],
+    ["Import {0} Items", "Importa {0} elementi"],
+    ["This file is protected by a password.", "Questo file é protetto da una password."],
+    ["Wrong Password", "Password Sbagliata"],
+    [
+        "The number of imported items exceeds your remaining quota. Upgrade to Premium to get unlimited items for your private vault!",
+        "Il numero di elementi importati supera la quota rimanente. Passa a Premium per ottenere elementi illimitati per il tuo archivio privato!"
+    ],
+    ["The number of imported items exceeds your remaining quota.", "Il numero di elementi importati supera la quota rimanente."],
+    ["Successfully imported {0} items.", "{0} Elementi importati con successo."],
+    ["This invite has expired", "Questo invito è scaduto"],
+    ["Accepted", "Accettato"],
+    ["expires {0}", "Scade {0}"],
+    ["Confirm Membership", "Confermare l'adesione"],
+    ["Organization Invite", "Invito Organizzazione"],
+    ["Please wait for an admin to complete the process. You will be notified as soon as you receive access.", "Attendere che un amministratore completi il processo. Riceverai una notifica non appena riceverai l'accesso."],
+    ["Enter Confirmation Code", ""],
```

Comment:
@MaKleSoft I haven't confirmed, but will this become an empty string, or pick up the English version?

Reply:
It will default to the English version.

Reply:
I missed that sorry 😅